lapis/job.py

```diff
@@ -87,12 +92,25 @@ def waiting_time(self) -> float:
             return self.in_queue_until - self.in_queue_since
         return float("Inf")
 
+    def modified_walltime(self):
+        if self.fileprovider and self.fileprovider.provides_all_files(self):
+            return walltime_models["maxeff"](self)
+        else:
+            return self.walltime
```

Comment:
You could rename the property `walltime` to `_walltime` and introduce 

```python
@property
def walltime(self):
    ...
```

In this method you could then implement the actual modification.

Reply:
Good point. I changed this but sadly I can't commit it because I have one file that doesn't start with inputs but with a function. black keeps deleting the 2 blank lines before the function while flake8 complains that there are 2 blank lines missing...:D